apis/nucleus/src/utils/style/styling-props.js

```diff
@@ -51,8 +51,9 @@ function getBackgroundPosition(bgComp) {
 
 function getBackgroundSize(bgComp) {
   let bkgImageSize = imageSizingToCssProperty.originalSize;
-  if (bgComp?.bgImage?.sizing) {
-    bkgImageSize = imageSizingToCssProperty[bgComp.bgImage.sizing];
+  const size = bgComp?.bgImage?.sizing || bgComp?.bgImage?.size;
```

Comment:
It's a little annoying that the api differs here and that we have to support two cases like this.
Don't know if it's "too late" to change or we have good reason to add `size` as well.

Reply:
Yeah, I acquired this from another place (Action button) which used these keys.

Reply:
It's not too late to change. I can change to what Nebula used before.